diff --git a/src/SettingsDaemon.vala b/src/SettingsDaemon.vala
index 21209ab..38c4711 100644
--- a/src/SettingsDaemon.vala
+++ b/src/SettingsDaemon.vala
@@ -26,24 +26,16 @@ public class SettingsDaemon : Object {
                               "org.gnome.settings-daemon.plugins.clipboard",
                               "org.gnome.settings-daemon.plugins.font",
                               "org.gnome.settings-daemon.plugins.gconf",
-                              "org.gnome.settings-daemon.plugins.gsdwacom",
-                              "org.gnome.settings-daemon.plugins.housekeeping",
                               "org.gnome.settings-daemon.plugins.keybindings",
                               "org.gnome.settings-daemon.plugins.keyboard",
-                              "org.gnome.settings-daemon.plugins.media-keys",
                               "org.gnome.settings-daemon.plugins.mouse",
                               "org.gnome.settings-daemon.plugins.print-notifications",
-                              "org.gnome.settings-daemon.plugins.smartcard",
-                              "org.gnome.settings-daemon.plugins.sound",
-                              "org.gnome.settings-daemon.plugins.wacom" };
+                              "org.gnome.settings-daemon.plugins.sound" };
 
         string[] enabled =  { "org.gnome.settings-daemon.plugins.a11y-keyboard",
                               "org.gnome.settings-daemon.plugins.a11y-settings",
-                              "org.gnome.settings-daemon.plugins.color",
                               "org.gnome.settings-daemon.plugins.cursor",
-                              "org.gnome.settings-daemon.plugins.power",
-                              "org.gnome.settings-daemon.plugins.xrandr",
-                              "org.gnome.settings-daemon.plugins.xsettings" };
+                              "org.gnome.settings-daemon.plugins.xrandr"};
 
         foreach (var schema in disabled) {
             set_plugin_enabled (schema, false);
